src/main/kotlin/com/hibob/academy/employeeFeedback/model/models.kt

```diff
@@ -0,0 +1,84 @@
+package com.hibob.academy.employeeFeedback.model
+
+import com.hibob.nullability.Department
+import java.time.LocalDate
+
+// Enum
+
+enum class FeedbackStatus {
+    UNREVIEWED,
+    REVIEWED
+}
+
+enum class Department {
+    HR,
+    OPERATIONS,
+    DEVELOPMENT,
+    MANAGEMENT
+}
+
+enum class Role {
+    EMPLOYEE,
+    HR,
+    ADMIN
+}
+
+// Data class
+
+data class Feedback(
+    val id: Long,
+    val senderId: Long?,
+    val text: String,
+    val isAnonymous: Boolean,
+    val status: FeedbackStatus,
+    val createdAt: LocalDate,
+    val lastModifiedAt: LocalDate
+)
+
+data class Respond(
+    val id: Long,
+    val feedbackId: Long,
+    val text: String,
+    val createdAt: LocalDate,
+    val lastModifiedAt: LocalDate
+)
+
+data class LoggedInUser(
+    val id: Long,
+    val companyId: Long,
+    val role: Role
```

Comment:
Just notice where you take the role from. I would consider split it to loggedIn user (id and companyId), and then another data class that has the loogedIn user + Role. You can call it LoggedInWithRole.